pkg/resourceinterpreter/configurableinterpreter/configmanager/manager.go

```diff
@@ -82,17 +82,25 @@ func (configManager *interpreterConfigManager) updateConfiguration() {
 		utilruntime.HandleError(fmt.Errorf("error updating configuration: %v", err))
 		return
 	}
-	configs := make(map[schema.GroupVersionKind]CustomAccessor, len(configurations))
 
+	accessors := make(map[schema.GroupVersionKind]CustomAccessor)
 	for _, c := range configurations {
```

Comment:
I think `configurations` need to be sorted.

Reply:
What's the use of storing it?

Reply:
Sorry, I misunderstand. I get your meaning.